app/models/question.rb

```diff
@@ -0,0 +1,14 @@
+class Question
+  ORDERED_KEYS = [:age, :name, :methods, :recording, :focus, :researcher, :incentive]
+  PARAM_KEYS = [:age, :participant_name, :guardian_name]
+
+  def self.first
+    ORDERED_KEYS.first
+  end
+
+  def self.next(current)
+    current_index = ORDERED_KEYS.index(current.to_sym)
+    return nil if current_index.nil?
+    ORDERED_KEYS[current_index.next]
+  end
+end
```

Comment:
Apologies for the bad line endings, I hadn't rebased onto your `.editorconfig` stuff. Will fix and `push -f`